nraufu/portofolio
# #30 tests: testing all endpoints

> #### What does this PR do?
- it tests all endpoints
> #### Description of Task to be completed
- add testing package (mocha, chai, chai-HTTP)
- test all endpoints
> #### How should this be manually tested?
- git clone the application URL(`https://github.com/nraufu/portofolio.git`)
- run `npm install` then run `npm run test`
- or run `npm run coverage` to see test coverage percentage
> #### What are the relevant Trello stories?
- https://trello.com/c/U0vxcpt9/30-test-all-endpoints-with-mocha
> #### Screenshots (if appropriate)
![tests](https://user-images.githubusercontent.com/49249449/90666088-32270a80-e24d-11ea-82c8-45728116aa28.PNG)
